### PR TITLE
Change updateDomainPassword method using from "Patch"to "Post"

### PR DESCRIPTION
**Pull request by Emma0727**
Change odata action "updateDomainPassword " method from patch to post.
We update public doc to notify the developer who already use this API to switch to use POST. And we will support both PATCH and POST until July to give customer some time to switch to use POST. Finally, we will switch to POST by the end of June.

Roadmap id: 14072

### CODE REVIEW

File: api-reference/beta/includes/snippets/objc/cloudpconpremisesconnection-updateaddomainpassword-objc-snippets.md [hunk 1]
```diff
@@ -8,7 +8,7 @@ MSHTTPClient *httpClient = [MSClientFactory createHTTPClientWithAuthenticationPr
 
 NSString *MSGraphBaseURL = @"https://graph.microsoft.com/beta/";
 NSMutableURLRequest *urlRequest = [NSMutableURLRequest requestWithURL:[NSURL URLWithString:[MSGraphBaseURL stringByAppendingString:@"/deviceManagement/virtualEndpoint/onPremisesConnections/{Id}/UpdateAdDomainPassword"]]];
-[urlRequest setHTTPMethod:@"PATCH"];
```
Comment (RuiHou105) [inline api-reference/beta/includes/snippets/objc/cloudpconpremisesconnection-updateaddomainpassword-objc-snippets.md:11]: No need to update these snippet files. Actually these files are generated/updated automatically by their tools.

**Review by RuiHou105 — CHANGES_REQUESTED** [review]
1.please resolve comments. Thanks.

2.Please create a roadmap workitem and then attach it into the description. Here is an example https://microsoftgraph.visualstudio.com/Roadmap/_workitems/edit/12535/. If you don't have access, please follow these steps to get access first:

Request access via [Improved ADO Access - 1ES Wikirequest to join](https://www.1eswiki.com/wiki/Improved_ADO_Access) .
Join the Microsoft Graph Roadmap Users ([graphroadmap@microsoft.com](mailto:graphroadmap@microsoft.com)) security group on [idweb](https://idwebelements/GroupManagement.aspx?Group=graphroadmap&Operation=join) .

File: api-reference/beta/includes/snippets/objc/cloudpconpremisesconnection-updateaddomainpassword-objc-snippets.md (hunk 1 above)
Comment (Emma0727) [reply api-reference/beta/includes/snippets/objc/cloudpconpremisesconnection-updateaddomainpassword-objc-snippets.md]: thanks, just revert the change in snappet file.

File: api-reference/beta/api/cloudpconpremisesconnection-updateaddomainpassword.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ One of the following permissions is required to call this API. To learn more, in
 }
 -->
```
Comment (AshleyYangSZ) [inline api-reference/beta/api/cloudpconpremisesconnection-updateaddomainpassword.md:33]: Please add notification here to indicate that this API will support both POST and PATCH for a month, and will finally switch to PATCH by the end of June.

Comment (Emma0727) [reply api-reference/beta/api/cloudpconpremisesconnection-updateaddomainpassword.md:35]: Added

**Review by RuiHou105 — APPROVED** [review]

**Comment by RuiHou105** [comment]
@Lauragra  @JarbasHorst Could you please help review this PR ?  Thanks!

**Comment by isvargasmsft** [comment]
Hello @Emma0727, @RuiHou105, @AshleyYangSZ; this scenario qualifies as a deprecation. Please remove the custom note and follow the guidelines for deprecation here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211775/Deprecate-docs?anchor=deprecate-content-for-a-method

**Review by isvargasmsft — CHANGES_REQUESTED** [review]
Please follow the deprecation process here: https://dev.azure.com/msazure/One/_wiki/wikis/Microsoft%20Graph%20Partners/211775/Deprecate-docs?anchor=deprecate-content-for-a-method

**Review by AshleyYangSZ — APPROVED** [review]

**Review by isvargasmsft — APPROVED** [review]
@Lauragra, would you be so kind to take a look at the changelog entry for this deprecation, please? 

I just want to make sure it's appropriate. 

Kind regards, 

Isaac

File: api-reference/beta/api/cloudpconpremisesconnection-updateaddomainpassword.md [hunk 3]
```diff
@@ -27,6 +27,17 @@ One of the following permissions is required to call this API. To learn more, in
...
 
+> [!CAUTION] 
+> Use POST for this operation. The PATCH method for this API will be deprecated after June 2022.
+
+<!-- {
+  "blockType": "ignored"
+}
+-->
+``` http
+POST /deviceManagement/virtualEndpoint/onPremisesConnections/{Id}/UpdateAdDomainPassword
```
Comment (Lauragra) [inline api-reference/beta/api/cloudpconpremisesconnection-updateaddomainpassword.md:38]: To be clear, I think we should keep both the POST and PATCH syntax in the topic until the deprecation period is over, but call out that the PATCH syntax is deprecated.

File: api-reference/beta/api/cloudpconpremisesconnection-updateaddomainpassword.md [hunk 4]
```diff
@@ -27,6 +27,17 @@ One of the following permissions is required to call this API. To learn more, in
 
 ## HTTP request
 
+> [!CAUTION] 
+> Use POST for this operation. The PATCH method for this API will be deprecated after June 2022.
```
Comment (Lauragra) [inline api-reference/beta/api/cloudpconpremisesconnection-updateaddomainpassword.md:31]: I suggest revising this note as follows:
The PATCH syntax for this action is deprecated and will stop working after June 2022. We recommend that you use POST going forward.

**Comment by isvargasmsft** [comment]
@Emma0727 I took the liberty of implementing Laura's feedback. 

Please take a look and add the label "ready to merge" again if you are ok with the changes.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**